package.json

```diff
@@ -32,8 +32,8 @@
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
     "prepublishOnly": "yarn setup && yarn build:clean && yarn lint && yarn test:nobuild",
     "setup": "yarn install && yarn allow-scripts",
-    "test": "yarn build && yarn test:nobuild",
-    "test:nobuild": "node test/index.js"
+    "test": "yarn test:nobuild",
```

Comment:
Can we collapse these into one task? Additionally according to the template there are usually two tasks, `test` and `test:watch`.